style.css

```diff
@@ -0,0 +1,124 @@
+:root {
+  --d-green: #253d2c;
+  --l-green: #68ba7f;
+  --dark: #000000;
+  --white: #ffffff;
+  --grey: #d9d9d9;
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+i {
```

Comment:
is it really a default style. addding padding to all the `i` tags. what if other `i` tag doesn't require padding